src/lib.rs

```diff
@@ -26,6 +26,12 @@ impl std::fmt::Display for Error {
     }
 }
 
+impl std::error::Error for Error {
+    fn description(&self) -> &str {
```

Comment:
[std::error::Error::description](https://doc.rust-lang.org/std/error/trait.Error.html#method.description) is soft-deprecated, impl `Display` is enough.

Reply:
This is required for compability with failure. OK then, I'll fix this in my code by wrapping lxc::Error into custom type which directly implements failure::Fail.

Reply:
This is just the `description` function implementation was deprecated. Implementing `Error` is a good idea. You can reopen this PR and amend your commit to remove the unnecessary code.